production/docker-compose.yml

```diff
@@ -28,7 +28,7 @@ services:
     image: mariadb:10.10.2-jammy
     restart: unless-stopped
     volumes:
-      - database:/var/lib/mysql
+      - data:/var/lib/mysql
```

Comment:
This is the same volume name as the MSSQL database container. I suggest you change this volume name to for example `landingpage-database` and also change the volume name of app-database-prd to something along the lines of `app-database`. 
Don't forget to change the top-level volumes-section in this file accordingly.